pkg/k8shandler/defaults.go

```diff
@@ -22,6 +22,8 @@ const (
 	elasticsearchCertsPath  = "/etc/openshift/elasticsearch/secret"
 	elasticsearchConfigPath = "/usr/share/java/elasticsearch/config"
 	heapDumpLocation        = "/elasticsearch/persistent/heapdump.hprof"
+
+	k8sTokenFile = "/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
formatting

Reply:
this is after `gofmt` its because of the whitespace between the lines. 
https://github.com/openshift/elasticsearch-operator/pull/157/files#diff-d2cc20d21811e6ef5a9fa7472806cbf0R20 is the same situation